package.json

```diff
@@ -15,18 +15,19 @@
     "postinstall": "lerna bootstrap",
     "prepublish": "lerna run clean --yes && lerna run build",
     "publish-packages": "lerna publish --exact --conventional-commits --registry=https://registry.npmjs.org/",
-    "report-coverage": "cat ./coverage/lcov.info | ./node_modules/.bin/coveralls",
     "storybook": "start-storybook -p 6006 -c .storybook -s ./.storybook/dist",
     "test": "./node_modules/.bin/jest --coverage",
     "test:ci": "yarn run test --maxWorkers=2 && yarn run report-coverage",
-    "test:coverage": "yarn run test && yarn run report-coverage"
+    "test:coverage": "yarn run test && ./node_modules/codecov/bin/codecov -t 9729bc93-20e5-42cb-aa55-a5308bd88f52",
```

Comment:
We'll want to pull the codecov token out of here and read from an environment variable.

Reply:
why? the coverage reports are only uploaded if someone runs coverage command. 

if react-ui is oss, shouldn't anyone be able to generate a codecov diff?

Reply:
hmm this is interesting https://github.com/codecov/codecov-node/issues/64